pyHS100/pyHS100.py

```diff
@@ -291,8 +291,12 @@ def location(self):
         :rtype: dict
         """
         info = self.sys_info
-        return {"latitude": info["latitude"],
-                "longitude": info["longitude"]}
+        if ("latitude" in info and "longitude" in info):
+            return {"latitude": info["latitude"],
+                    "longitude": info["longitude"]}
+        else:
+            return {"latitude": "N/A",
+                    "longitude": "N/A"}
```

Comment:
Agree, `None` is the appropriate Python construct, rather than a string where one is expecting a number.